src/metabase/automagic_dashboards/core.clj

```diff
@@ -190,7 +190,7 @@
   (let [table (->> metric :table_id (t2/select-one Table :id))]
     {:entity       metric
      :full-name    (if (:id metric)
-                     (tru "{0} metric" (:name metric))
+                     (deferred-trun "{0} metric" "{0} metrics" (:name metric))
```

Comment:
For whatever reason, If I had used `trun` instead `deferred-trun`, unfortunately the plural message (`message-pl`) was not outputted. I'm not sure if it's the deficiency in the `trun` macro or if the issue lies somewhere else.

Reply:
This is a good question for @metabase/backend-developers

Reply:
I tried it with `trun` and I still saw the `msgid_plural` being included in metabase.pot, although the script did also log `msgcat: msgid '{0} metric' is used without plural and with plural.` Not sure if that's different from what you're saying @ariya

Reply:
@noahmoss that `msgid_plural` likely was originated from the front-end message catalog, since `metabase.pot` is the merge of both backend and frontend message catalogs.

Can you check `metabase_backend.pot` instead? That one is exclusively only for the backend strings.

Reply:
@ariya Sorry for the delay, this fell off my radar. Just looked into it and it turns out there was another simple typo in the i18n build script, so it wasn't looking for the right macro name. (And this is the first time anyone has tried to use `trun`). Will put up a separate PR to fix, thanks for catching

Reply:
Merged https://github.com/metabase/metabase/pull/31861